.github/workflows/ci-gh-docs.yml

```diff
@@ -17,9 +13,11 @@ env:
   REPO_URL: ${{ github.event.pull_request.head.repo.html_url || github.event.repository.html_url }}
   # Prevent output buffering
   PYTHONUNBUFFERED: 1
+  ARTIFACTS_DIR: "${{ github.workspace }}/.artifacts"
+  ARTIFACT_NAME: "legate-core-docs"
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: linux-amd64-cpu32
```

Comment:
Can you set this with the fallback to ubnutu-latest? e.g. the other workflows have `${{ contains(github.repository, 'nv-legate/legate.core') && 'linux-amd64-32cpu' || 'ubuntu-latest' }}`

https://github.com/nv-legate/legate.core/blob/branch-24.01/.github/workflows/ci-gh.yml#L20C27-L20C128